opendrift/__init__.py

```diff
@@ -79,7 +79,12 @@ def open(filename):
     return o
 
 def versions():
-    from importlib.metadata import version
+    def version(package):
+        try:
+            return importlib.metadata.version(package)
+        except:
```

Comment:
Can you filter on a specific exception here? What is the reason that you run into this. Normally this works when either in a git repository or installed setting.

Reply:
So the reason is that `importlib` isn't available?

Reply:
No, `adios_db`